t/config-center-yaml/plugin-metadata.t

```diff
@@ -33,8 +33,8 @@ _EOC_
 
     $block->set_value("yaml_config", $yaml_config);
 
-    if (!$block->no_error_log) {
-        $block->set_value("no_error_log", "[error]");
+    if (!$block->no_error_log && !$block->error_log) {
+            $block->set_value("no_error_log", "[error]");
```

Comment:
Sorry for my mistake. Please use a 4-spaces indentation.

Reply:
The codelint is powerless

Reply:
I'm not notice this.